recipes/xilinx-runtime/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/env/bash
+set -e
+
+mkdir -p $SRC_DIR/build-release
+
+cmake \
+  -S $SRC_DIR/src  \
+  -B $SRC_DIR/build-release  \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_PREFIX_PATH=${PREFIX} \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DCMAKE_SKIP_INSTALL_RPATH=ON
+
+cmake --build $SRC_DIR/build-release --parallel ${CPU_COUNT}
+
+cmake --install $SRC_DIR/build-release
+
+# XILINX_XRT
+mkdir -p $PREFIX/etc/conda/activate.d
+echo -e "#!/bin/env/bash\nexport XILINX_XRT=\$CONDA_PREFIX\n" > $PREFIX/etc/conda/activate.d/env_vars.sh
```

Comment:
@timsnyder ok so what name should I use? `xilinx_runtime_xilinx_xrt.sh`

I like this 1 liner approach, and I'm sure it can be duplicated for the deactivate.

Reply:
```suggestion
echo -e "#!/bin/env/bash\nexport XILINX_XRT=\$CONDA_PREFIX\n" > $PREFIX/etc/conda/activate.d/${PKG_NAME}_activate.sh
```